benches/circuit.rs

```diff
@@ -6,6 +6,7 @@ use criterion::{BenchmarkId, Criterion};
 #[cfg(unix)]
 use pprof::criterion::{Output, PProfProfiler};
 
+use orchard::note::NoteType;
```

Comment:
Consider renaming to `NoteAsset` ("type" is a very overloaded term).

Reply:
We renamed it to be `orchard::note::AssetId` which seems to be more aligned with the spec.